seminar/feedpage/views.py

```diff
@@ -0,0 +1,45 @@
+from django.shortcuts import render
+from .models import Feed
+from django.shortcuts import redirect
+
+# Create your views here.
+
+def index(request):
+    if request.method == 'GET':
+        feeds = Feed.objects.all()
+        return render(request, 'feedpage/index.html', {'feeds': feeds})
+    elif request.method == 'POST':
+        title = request.POST['title']
+        content = request.POST['content']
+        Feed.objects.create(title=title, content=content)
+        return redirect('/feeds')
+
+
+def new(request):
+    return render(request, 'feedpage/new.html')
+
+
+def show(request, id):
+    feed = Feed.objects.get(id=id)
+    return render(request, 'feedpage/show.html', {'feed':feed})
+
+
+def edit(request, id):
+    feed = Feed.objects.get(id=id)
+    return render(request, 'feedpage/edit.html', {'feed':feed})
+
+
+def update(request, id):
+    feed = Feed.objects.get(id=id)
+    feed.title = request.POST['title']
+    feed.content = request.POST['content']
+    feed.save()
+    return render(request, 'feedpage/show.html', {'feed':feed})
```

Comment:
save 함수를 써도 좋지만 update 도 있다는 것을 참고해주세요~

```suggestion
    title = request.POST['title']
    content = request.POST['content']
    Feed.objects.update(id=id, title=title, content=content)
    return redirect("/show/%d" %id)
```